### ISSUE TITLE
Doom emacs-like `space space` directory navigator

### ISSUE THREAD

**Issue by CBenoit** [helix-editor/helix#361]
> The other idea I have is sort of `space space` in doom emacs which we can navigate between directories easily and show only the current directory so that is at least easier compared to `:e xxx/<tab>`.

_Originally posted by @pickfire in https://github.com/helix-editor/helix/pull/336#issuecomment-866448160_

This is useful when the name is not known but we have an idea of it's location.

Mapping would probably be something else than `space space`.

**Comment by CBenoit** [comment]
> EDIT by @pickfire: `space space` in doom emacs is `space f` in helix now. So `space f` in doom emacs we need to figure out a key for it, I wish we could use `space space` for something so common.

@pickfire I think you could remap that?

**Comment by gbaranski** [comment]
I personally use space-space for switching to last file.

**Comment by pickfire** [comment]
> @pickfire I think you could remap that?

Yes, I could. But I wonder if the current `space space` is very much used that we need to make it `space space` compared to the one in doom emacs, or is there any other keys that fit it better, like escape or backticks.

**Comment by Cons-Cat** [comment]
Personally I find `space p f` for searching within the current project generally more useful than `space space`
